src/google/protobuf/util/json_util_test.cc

```diff
@@ -241,6 +242,25 @@ TEST_F(JsonUtilTest, TestPrintEnumsAsIntsWithDefaultValue) {
   EXPECT_EQ(proto3::BAR, parsed.enum_value3());
 }
 
+TEST_F(JsonUtilTest, TestPrintEnumsAsStringWithDefaultValue) {
```

Comment:
It seems to me like the name of this test is a little bit inaccurate since it's not trying to print enums as strings--maybe rename it `TestPrintProto2EnumAsIntWithDefaultValue`.

Reply:
done